packages/editor/src/browser/editor-widget-factory.ts

```diff
@@ -50,7 +50,7 @@ export class EditorWidgetFactory implements WidgetFactory {
             newEditor.title.closable = true;
             newEditor.title.label = this.labelProvider.getName(uri);
             newEditor.title.iconClass = icon + ' file-icon';
-            newEditor.title.caption = this.labelProvider.getLongName(uri);
```

Comment:
but it breaks single root scenarios? Should not we work on improving `getLongName` for multi-roots?

Have you checked what VS Code show? I would imagine we can show something like `README.md (cpp)`?